File: WisolSMTLineApp/WisolSMTLineApp/Model/ProductionPlan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WisolSMTLineApp.Model
{
    public class ProductionPlan
    {
        public string WorkingDate { set; get; }
        public int FactoryID { set; get; }
        public int LineID { set; get; }
        public int ShiftID { set; get; }
        public int ProductID { set; get; }
        public string ProductName { set; get; }
        public int OrderedQty { set; get; }
        public int GoodProdQty { set; get; }
        public int NGProdQty { set; get; }
        public bool Disabled { set; get; }
    }
}

File: src/models/schemes/production_plan.js
var { sequelize, Sequelize } = require("./db");
const ProductionPlan = sequelize.define(
	"sf_production_plan",
	{
		working_date: {
			type: Sequelize.DATEONLY,
			primaryKey: true,
		},
		factory_id: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			primaryKey: true,
			references: {
				model: "sf_factories",
				key: "id",
			},
			onDelete: "CASCADE",
			onUpdate: "CASCADE",
		},
		shift_id: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			primaryKey: true,
			references: {
				model: "sf_shift",
				key: "id",
			},
			onDelete: "CASCADE",
			onUpdate: "CASCADE",
		},
		line_id: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			primaryKey: true,
			references: {
				model: "sf_prod_lines",
				key: "id",
			},
			onDelete: "CASCADE",
			onUpdate: "CASCADE",
		},
		product_id: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			primaryKey: true,
			references: {
				model: "sf_products",
				key: "id",
			},
			onDelete: "CASCADE",
			onUpdate: "CASCADE",
		},
		ordered_qty: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			defaultValue: 0,
		},
		good_prod_qty: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			defaultValue: 0,
		},
		remain_qty: {
			type: Sequelize.INTEGER(11).UNSIGNED,
			defaultValue: 0,
		},
		is_active: {
			type: Sequelize.BOOLEAN,
			allowNull: false,
			defaultValue: true,
		},
	},
	{
		tableName: "sf_production_plan",
		timestamps: true,
		createdAt: "created_at",
		updatedAt: "updated_at",
		charset: "utf8mb4",
		collate: "utf8mb4_unicode_ci",
		underscored: true,
	}
);
module.exports = ProductionPlan;
